user_docs/en/userGuide.t2t

```diff
@@ -346,10 +346,10 @@ When within a table, the following key commands are also available:
 | Move to last column | control+alt+end | Moves the system caret to the last column (staying in the same row) |
 | Move to first row | control+alt+pageUp | Moves the system caret to the first row (staying in the same column) |
 | Move to last row | control+alt+pageDown | Moves the system caret to the last row (staying in the same column) |
-| Say all current column | NVDA+control+alt+DownArrow | Reads column vertically from current table cell down to the last cell in the column. |
-| Say all current row | NVDA+control+alt+RightArrow | Reads row horizontally from current table cell up untill the last cell in the row. |
-| Speak current column | NVDA+control+alt+UpArrow | Reads current column vertically from top to bottom without moving system caret. |
-| Speak current row | NVDA+control+alt+LeftArrow | Reads current row vertically from left to right without moving system caret. |
+| Say current column from current cell down | NVDA+control+alt+DownArrow | Reads column vertically from current table cell down to the last cell in the column. |

```

Comment:
I think a short name is better here, with more information in the description. I've also fixed the formatting of the key commands.

```suggestion
| Speak column downwards | ``NVDA+control+alt+downArrow`` | Reads column vertically from the current table cell downwards to the last cell in the column. |
```